sources/Application/UI/Base/Component/Input/Set/Set.php

```diff
@@ -183,6 +188,30 @@ public function HasRemoveItemButton(): bool
 		return $this->bHasRemoveItemButton;
 	}
 
+	/**
+	 * SetAddOptionButtonJsOnClick.
+	 *
+	 * @param string $sJsOnClick
+	 *
+	 * @return $this
+	 */
+	public function SetAddOptionButtonJsOnClick(string $sJsOnClick): Set
```

Comment:
We don't put type hint on return as it will be an issue with inheritance. Will talk about this further IRL.
```suggestion
	public function SetAddOptionButtonJsOnClick(string $sJsOnClick)
```